### PR TITLE
hide rs behind feature flag

### PR DESCRIPTION
**Pull request by nvdk**

### CODE REVIEW

**Review by usrtim — APPROVED** [review]

**Review by elpoelma — CHANGES_REQUESTED** [review]
Works well overall, but it seems that the 'insert regulatory statement' functionality in an agendapoint is not yet disabled when needed.

**Comment by nvdk** [comment]
> Works well overall, but it seems that the 'insert regulatory statement' functionality in an agendapoint is not yet disabled when needed.

thanks for catching that!

File: app/instance-initializers/features-setup.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import config from 'frontend-gelinkt-notuleren/config/environment';
+
+export function initialize(appInstance) {
+  const features = appInstance.lookup('service:features');
+  const featureConfig = config.featureFlags || {};
+  for (const key of Object.keys(featureConfig)) {
+    const val = featureConfig[key];
+    if (typeof val === 'string' || val instanceof String) {
```
Comment (lagartoverde) [inline app/instance-initializers/features-setup.js:8]: Do we need both checks? Is there any string that is not typeof string but instanceof String?

**Review by lagartoverde — CHANGES_REQUESTED** [review]
Works well overall, as Elena said before me we also have to disable the regulatory statement plugin

File: app/instance-initializers/features-setup.js (hunk 1 above)
Comment (lagartoverde) [reply app/instance-initializers/features-setup.js:8]: ok, then good job for catching that :)

**Pull request merged**